src/components/AssetItem/AssetItem.js

```diff
@@ -135,33 +129,44 @@ class AssetItem extends PureComponent<Props, StateProps> {
               <Fragment>
                 <div
                   className='item -edit'
-                  onMouseEnter={this.handleMouseEnterEdit}
-                  onMouseLeave={this.handleMouseLeaveEdit}
                   onClick={this.handleClickEdit}
                 >
                   <JTooltip text={t`Edit`}>
                     <JIcon
-                      color={isHoveredEdit ? 'sky' : 'blue'}
                       name='edit'
                     />
                   </JTooltip>
                 </div>
-                <div
-                  className='item -delete'
-                  onMouseEnter={this.handleMouseEnterRemove}
-                  onMouseLeave={this.handleMouseLeaveRemove}
-                  onClick={this.handleMouseLeaveRemove}
-                >
-                  <ButtonWithConfirm
```

Comment:
why you decided to remove ButtonWithConfirm and place its code here?

Reply:
I think we should to remove ButtonWIthConfirm from codebase.